src/Transports/MassTransit.KafkaIntegration/ITopicNameFormatter.cs

```diff
@@ -0,0 +1,26 @@
+namespace MassTransit.KafkaIntegration
+{
+    public interface ITopicNameFormatter
+    {
+        string GetTopicName<TKey, TValue>()
+            where TValue : class;
+    }
+
+
+    public class DefaultTopicNameFormatter :
+        ITopicNameFormatter
+    {
+        readonly string _topic;
+
+        public DefaultTopicNameFormatter(string topic)
```

Comment:
So use the same topic for everything?

Reply:
it should be called `ConstantTopicNameFormatter` it is used for extensions
it was in my todo list, but I've forgot